moor_generator/lib/src/writer/tables/table_writer.dart

```diff
@@ -365,7 +365,8 @@ class TableWriter extends TableOrViewWriter {
 
       buffer
         ..write('@override\n')
-        ..write('List<String> get customConstraints => const [$value];\n');
+        ..write('List<String> get customConstraints => super.customConstraints'
```

Comment:
Similar here, it looks like this change is not necessary?

Reply:
That's necessary because foreign key constraints override the custom constraints that you provide in the definition class. With super the custom constraints is merged with foreign key constraints.